### DISCUSSION TITLE
Some Trash on Github ( Unused repositories )

### DISCUSSION THREAD

**Discussion by DSDmark** [community/community#28132] [Other Feature Feedback, Questions, & Ideas] [answered]
Hey guys,

when I was finding for projects. most of the time I had seen the unused repositories and profile 
which is trash nothing more but still on Github.

There are two types of repositories

1:- made by some other user but never used.

2:- I make it by myself but somehow forgot the password or some details ( that is in my case )

* some GitHub profile is even 3 or more years old and is no longer used but are still on Github

so, my question is, how can I request to delete that kind of unused repositories that are no longer used?

**Comment by jfullstackdev** [comment, marked as answer]
if that is from another user, I doubt, there were several discussions about name squatting and releasing of names because they are not being active anymore,

as for your own projects, you can simply delete or archive it

**Reply by DSDmark** [reply]
its make sense

**Reply by MrKrishnaAgarwal** [reply]
> its make sence

*sense

**Comment by woeterman94** [comment]
Only owners of repositories can delete the repository. 
If you lost access to your account try to open a support ticket here: https://github.com/contact
